includes/admin/class-wc-stripe-privacy.php

```diff
@@ -90,7 +90,7 @@ protected function get_stripe_orders( $email_address, $page ) {
 	 */
 	public function get_privacy_message() {
 		/* translators: %s URL to docs */
-		return wpautop( sprintf( __( 'By using this extension, you may be storing personal data or sharing data with an external service. <a href="%s" target="_blank">Learn more about how this works, including what you may want to include in your privacy policy.</a>', 'woocommerce-gateway-stripe' ), 'https://docs.woocommerce.com/document/privacy-payments/#woocommerce-gateway-stripe' ) );
+		return wpautop( sprintf( __( 'By using this extension, you may be storing personal data or sharing data with an external service. <a href="%s" target="_blank">Learn more about how this works, including what you may want to include in your privacy policy.</a>', 'woocommerce-gateway-stripe' ), 'https://woocommerce.com/document/privacy-payments/#section-3' ) );
```

Comment:
This link (https://woocommerce.com/document/privacy-payments/#section-3) may be prone to change, soon. For more info, [see the following comment](https://github.com/woocommerce/woocommerce-gateway-stripe/issues/2149#issuecomment-969299485).